src/components/translator/formTranslation.ts

```diff
@@ -0,0 +1,53 @@
+import htsPT from './hts_pre_test_DRAFT.PT.json';
```

Comment:
I believe you had these HTS forms for testing purposes when you were developing but I don't think they should be added to the repo. Please remove